content/getting-started/sections/installing-atom.md

```diff
@@ -78,7 +78,7 @@ To install Atom on Debian, Ubuntu, or related distributions, add our official
 package repository to your system by running the following commands:
 
 ``` command-line
-$ wget -qO - https://packagecloud.io/AtomEditor/atom/gpgkey | sudo apt-key add -
+$ wget -qO - https://packagecloud.io/AtomEditor/atom/gpgkey | gpg --dearmor | sudo tee /usr/share/keyrings/atom-archive-keyring.gpg
 $ sudo sh -c 'echo "deb [arch=amd64] https://packagecloud.io/AtomEditor/atom/any/ any main" > /etc/apt/sources.list.d/atom.list'
```

Comment:
The `[arch=amd64]` part would also need to be updated to `[signed-by=/usr/share/keyrings/atom-archive-keyring.gpg arch=amd64]` if you went with this approach.

```sh
 $ sudo sh -c 'echo "deb [signed-by=/usr/share/keyrings/atom-archive-keyring.gpg arch=amd64] https://packagecloud.io/AtomEditor/atom/any/ any main" > /etc/apt/sources.list.d/atom.list'
```

Reply:
Got it. Would you like me to add that in, or have you?

Reply:
Add it in yourself, I don't believe I have permissions to do anything to your commits.

Reply:
Done.